sprockets/logging.py

```diff
@@ -18,11 +18,12 @@
 import traceback
 
 try:
-    from tornado import log
-except ImportError:
+    from tornado import escape, log
+except ImportError:  # pragma no cover
+    escape = None
```

Comment:
Nvm on modifying *setup.py*.  Tornado isn't required so you can use everything but the `tornado_log_function` without having it installed.